src/modules/Projects/components/ProjectList/ProjectList.jsx

```diff
@@ -1,42 +1,71 @@
 import axios from "axios";
-import { useNavigate } from "react-router-dom";
-import { baseurl } from "../../../BaseUrls/BaseUrls";
+import { baseurl, RequestHeaders } from "../../../BaseUrls/BaseUrls";
 import { useEffect, useState } from "react";
+import { toast } from "react-toastify";
+
+import Button from "react-bootstrap/Button";
+import Modal from "react-bootstrap/Modal";
+
+import { useNavigate } from "react-router-dom";
 
 export default function ProjectList() {
   const [projectList, setProjectList] = useState([]);
-  let navigate = useNavigate();
-  let getProducts = async () => {
+  const [id, setId] = useState();
+  const [show, setShow] = useState(false);
+  const navigate = useNavigate();
+  const handleClose = () => setShow(false);
+  const handleShow = (id) => {
+    setId(id);
+    setShow(true);
+  };
+
+  let delete_project = async () => {
     try {
-      let response = await axios.get(`${baseurl}/Project/manager`, {
-        headers: {
-          Authorization: `Bearer ${localStorage.getItem("token")}`,
-        },
-      });
+      let response = await axios.delete(
+        `${baseurl}/Project/${id}`,
+        RequestHeaders
+      );
+      console.log(response);
+      toast.success("Deleted Project Successfully");
+      handleClose();
+      getProjects();
+    } catch (error) {
+      console.log(error);
+    }
+  };
+  const getProjects = async () => {
```

Comment:
We need to make this function wrapped in a useCallabck to avoid creation in every render or we can declare  in the useEffect  if we are only using it inside the useEffect.